### DISCUSSION TITLE
220509 포키 인터뷰 피드백

### DISCUSSION THREAD

**Discussion by YJGwon** [woowacourse-study/2022-lv2-effective-java-interview#5] [General]
### 인터뷰어 -> 인터뷰이

> 점수를 매겨주셔도 좋고, 체크 형식으로 작성해주신 뒤 짧은 코멘트를 남겨주셔도 좋습니다. 해당 항목에 대해 자유롭게 의견을 남겨주세요.

- 말하기 태도, 어조가 적절했나요?
- 전달하고자 하는 바를 명료하게 전달했나요?
- 본인만의 주관이 잘 드러났나요?
- 책의 내용을 단순히 수용하지 않고 본인의 경험과 잘 연결지어 이해했나요?
- 모르는 부분에 대해 잘 대처했나요?

### 인터뷰이 -> 인터뷰어

> 인터뷰어로써의 역량을 키우려는 것이 아닌 저희 스터디의 원활한 진행을 위한 피드백입니다. 각 크루가 아닌 전반적인 진행 상황에 대해 의견을 남겨주세요!

- 사전에 공유한 내용이 잘 전달되었나요?
- 질문의 방향이 단순히 fact check가 아닌 의견을 끌어내는 방향이었나요?
- 생각해보지 못했던 부분에 대해 생각해볼 수 있었나요?
- 새로운 관점을 배울 수 있었나요?

**Comment by YJGwon** [comment]
- [X] 사전에 공유한 내용이 잘 전달되었나요?
- [X] 질문의 방향이 단순히 fact check가 아닌 의견을 끌어내는 방향이었나요?
- [X] 생각해보지 못했던 부분에 대해 생각해볼 수 있었나요?
- [X] 새로운 관점을 배울 수 있었나요?

### comment
다양한 상황을 예시로 들어서 질문해주셔서 많은 고민을 해볼 수 있었습니다.

**Comment by tco0427** [comment]
인터뷰어 -> 인터뷰이

- [x] 말하기 태도, 어조가 적절했나요? 
- [x] 전달하고자 하는 바를 명료하게 전달했나요?
    -> 전달하고자 하는 바를 명료하게 전달할 뿐 아니라, 질문자의 의도도 확실하게 파악했다.
- [x] 본인만의 주관이 잘 드러났나요?
- [x] 책의 내용을 단순히 수용하지 않고 본인의 경험과 잘 연결지어 이해했나요?
    -> 미션을 진행하면서 커스텀 예외에 대한 고민이 많이 느껴졌다.
- [x] 모르는 부분에 대해 잘 대처했나요?
    -> 고민해보지 못한 부분에 대해서, 확실하게 말해줘서 좋았다.

**Comment by Ohzzi** [comment]
- 말하기 태도, 어조가 적절했나요?
  - 어조가 또렷하고 자신감이 담겨 있습니다. 사전에 내용을 잘 숙지하고 있다는 느낌을 줄 수 있을 것 같아요.
- 전달하고자 하는 바를 명료하게 전달했나요?
  - 생각하는 바가 정확하게 느껴지게 전달했습니다.
- 본인만의 주관이 잘 드러났나요?
  - 네. Custom Exception을 지양하는 이유가 명확했고, 여러 가지 상황을 제시해도 그 이유가 크게 흔들리지 않았습니다. 그러면서도 어떤 지점 이후부터는 Custom Exception의 사용을 고려하겠다는 기준도 있어서 좋았습니다.
- 책의 내용을 단순히 수용하지 않고 본인의 경험과 잘 연결지어 이해했나요?
  - 시작부터 책의 내용을 대차게 까고 시작했습니다. 좋네요.
- 모르는 부분에 대해 잘 대처했나요?
  - 의도하지 않은 예외도 함께 처리된다는 지적이 들어왔을 때 해당 부분에 고려해보겠다는 의견으로 잘 넘어갔습니다.

**Comment by ulimy** [comment]
전체적으로 차분하고 일관된 톤으로 대답해주셔서 좋았습니다! 자신이 고려해보지 못한 부분에 대해 솔직하게 얘기해주신 점도 좋았습니다. 다만, 고려해보지 못했지만 지금 잠깐 고민해보고 답변 드리고싶다라고 말하는 등 조금 더 대답에 열정이 느껴진다면 좋을 것 같습니다. 또한 현재 상황에서는 쓰지 않지만 만약에 커진다면 어떻게 할 것인지를 대답하는 것이 질문의 의도를 표현하는 방식이라고 생각합니다.

**Comment by yeon-06** [comment]
- 말하기 태도, 어조가 적절했나요?  
👉 O. 어조나 손짓(?)이 적당했다.
- 전달하고자 하는 바를 명료하게 전달했나요?  
👉 O. 
- 본인만의 주관이 잘 드러났나요?  
👉 O. 상대방이 더 고민하고 있으면 보충 설명을 해주는 등의 태도 좋았음
- 책의 내용을 단순히 수용하지 않고 본인의 경험과 잘 연결지어 이해했나요? 
👉 답변에서 자신의 경험을 잘 녹여낸 것을 느꼈음  
- 모르는 부분에 대해 잘 대처했나요?  
👉 O. 질문을 이해하지 못했으면 다시 물어보거나 모르는 것이 있으면 거기까지는 고려해보지 않았다, 등 솔직히 말해주어 좋았음

했던 질문들
-> 사용자에게 보이는 오류 메시지는 어떻게 관리할 수 있을까?
-> CustomException은 언제 사용할 수 있다고 판단하는가?
-> 우연한 예외 상황을 어떻게 미리 예상해서 디테일한 메시지를 작성할 수 있을까?

**Comment by kyukong** [comment]
- 말하기 태도, 어조가 적절했나요?
예상하지 못한 질문이 나와도 말의 속도가 일정하고 차분하게 설명을 잘 해주었습니다. 

- 전달하고자 하는 바를 명료하게 전달했나요?
간결하게 포키의 의견을 전달하는 것이 너무 멋있었습니다~!

- 본인만의 주관이 잘 드러났나요?
처음 발표부터 마지막 답변까지 의견이 일관되었습니다.

- 책의 내용을 단순히 수용하지 않고 본인의 경험과 잘 연결지어 이해했나요?
아이템 선정 이유부터 포키의 경험이 잘 녹아들었습니다. 예외에 관련해서 오래 고민하신 것 같아 좋았습니다.

- 모르는 부분에 대해 잘 대처했나요?
생각하지 못했던 부분에 솔직하게 답변하는 모습이 좋았습니다.
